docs-src/concepts/what-is-a-data-converter.md

```diff
@@ -8,14 +8,12 @@ tags:
   - explanation
 ---
 
-A Data Converter is a Temporal SDK component that encodes and decodes data entering and exiting a Temporal Server.
-
-- TypeScript: [Data Converters](https://legacy-documentation-sdks.temporal.io/typescript/data-converters)
-- Go: [Create a custom Data Converter](https://legacy-documentation-sdks.temporal.io/go/how-to-create-a-custom-data-converter-in-go)
+A Data Converter is a Temporal SDK component that encodes and decodes data entering, stored on, and exiting a Temporal Cluster.
+It is used by the Temporal SDK framework to serialize/deserialize input and output of Activities and Workflows that need to be sent over the wire to the Temporal Cluster.
 
 ![Data Converter encodes and decodes data](/diagrams/default-data-converter.svg)
 
-Data is encoded before it is sent to a Temporal Server, and it is decoded when it is received from a Temporal Server.
+The Data Converter encodes all data from your application before it is sent to the Temporal Cluster in the Client call. When the Temporal Server sends the encoded data back to the Worker, the Data Converter decodes the data for processing within your application. This ensures that all your sensitive data exists in its original format only on hosts that you control.
```

Comment:
I would clean the usage of terms here.
When we talk about temporal we usually reserve encoding, decoding and codec for byte <-> byte.

It should be "The Data Converter is responsible to serialization" or "The Data Converter is responsible for serialization and encoding". Just encoding is not right according to how we use terms right now.

Reply:
Updated to
`A Data Converter is a Temporal SDK component that serializes and encodes data entering, stored on, and exiting a Temporal Cluster.`